ykman-gui/qml/InterfaceView.qml

```diff
@@ -32,7 +32,7 @@ ColumnLayout {
             name: qsTr("OATH")
         }]
 
-    Component.onCompleted: load()
+    StackView.onActivating: load()
```

Comment:
What does this change do?

Reply:
`Component.onCompleted` only triggers when the component is initially instantiated; `StackView.onActivating` triggers whenever the StackView navigates to this item. So this probably doesn't make any practical difference for `InterfaceView` since it doesn't have any child views, but it does for `OtpView` and `Fido2View`.